stanford_profile_helper.module

```diff
@@ -264,6 +264,16 @@ function stanford_profile_helper_preprocess_input__submit__paragraph_action(&$va
  * Implements hook_entity_field_access().
  */
 function stanford_profile_helper_entity_field_access($operation, FieldDefinitionInterface $field_definition, AccountInterface $account, FieldItemListInterface $items = NULL) {
+  if (
+    $field_definition->getName() == 'status' &&
+    $field_definition->getTargetEntityTypeId() == 'node' &&
+    $items &&
+    $items->getEntity()->id()
```

Comment:
I don't understand how this doesn't disable unpublishing on all nodes.  But I tested it locally and it does in fact only target home, 404, and 403.

Reply:
it doesn't truly disable the ability, it just hides the publishing checkbox. alot less likely to unpublish those pages.